### ISSUE TITLE
[4.1.0-rc.2] Categories are not displayed any more

### ISSUE THREAD

**Issue by manooweb** [WordPress/gutenberg#10993]
**Describe the bug**
Categories are not displayed any more because of an error return by the REST API which does not accept any more per_page parameter value with "-1"

**To Reproduce**

1. Go to 'Post > All posts'
2. Click on 'Edit' link on one of the posts
3. Open chrome developer for seeing what's going wrong
4. See categories are not displayed
5. See in the chrome developer tool bar why. See screenshot below with the error message

**Expected behavior**
Display categories 

**Screenshots**
With Gutenberg 4.1.0-rc.1 version => OK

![image](https://user-images.githubusercontent.com/1003778/47419752-25364e80-d77d-11e8-84f9-5959871a61a4.png)

With Gutenberg 4.1.0-rc.2 version => KO with the error message "per_page: "per_page must be between 1 (inclusive) and 100 (inclusive)"

![image](https://user-images.githubusercontent.com/1003778/47420426-b1954100-d77e-11e8-9279-9ae1177523e8.png)

**Desktop**
 - OS: windows 10
 - Browser chrome
 - Version 70

**Additional context**
- WordPress 4.9.8
- Gutenberg 4.1.0-rc.2

**Comment by danielbachhuber** [comment]
Hi @manooweb,

Thanks for the report.

> Categories are not displayed any more because of an error return by the REST API which does not accept any more per_page parameter value with "-1"

The usage of `per_page=-1` was removed in #10762. I tested against a local copy of rc2 with WP 4.9.8 and was unable to reproduce the report.

Is this a plain WordPress install or do you have other plugins running? Can you try force flushing your browser cache?

**Comment by manooweb** [comment]
Hi @danielbachhuber,
It is a plain WordPress 4.9.8 install on my own development machine where I'm doing and testing my developments. 
With chrome developer tool bar the "Disable cache" option in the "Network tab" is always checked.
I have some plugins activated as ours which we are developing.
I synchronize Gutenberg plugin directly from Github and build it by myself locally.

So, 
- I desactivated all the plugins except Gutenberg
- I emptied the cache manually
- I closed and reopened chrome
- I rebuilt correctly Gutenberg in development mode and then in production mode

That's ok now after rebuilding Gutenberg. Very strange behaviour.
I'm closing this issue.
